go-controller/go.mod

```diff
@@ -44,6 +44,7 @@ require (
 	k8s.io/client-go v0.20.0
 	k8s.io/klog v0.0.0-20181102134211-b9b56d5dfc92
 	k8s.io/klog/v2 v2.4.0
+	k8s.io/kubernetes v1.20.0
```

Comment:
We shouldn't vendor kubernetes. If there's any code you need from there, just duplicate it somewhere. (I seem to recall we already copied the tainting code somewhere...)

Reply:
Being done here: https://github.com/ovn-org/ovn-kubernetes/pull/2183